docs/en/operation/Metrics-System.md

```diff
@@ -120,9 +129,237 @@ Metrics in Alluxio have the following format for non-master node metrics:
+| Master.CapacityFree | Free capacity of the Alluxio file system in bytes |
+| Master.CapacityFreeTier<TIER> | Free capacity in tier <TIER> of the Alluxio file system in bytes |
+
+* Under storage capacity
+
+| Metric Name | Description |
+|-------------------------|--------------------------------------------------|
+| Master.UfsCapacityTotal | Total capacity of the under file system in bytes |
+| Master.UfsCapacityUsed | Used capacity of the under file system in bytes |
+| Master.UfsCapacityFree | Free capacity of the under file system in bytes |
+
+* Total amount of data transferred through Alluxio and I/O throughput estimates (meter statistics)
+
+| Metric Name | Description |
+|--------------------------------------|---------------------------------------------------------------------------|
+| cluster.BytesReadAlluxio | Total number of bytes read from Alluxio storage. This does not include UFS reads |
+| cluster.BytesReadAlluxioThroughput | Bytes read throughput from Alluxio storage |
+| cluster.BytesReadDomain | Total number of bytes read from Alluxio storage via domain socket |
+| cluster.BytesReadDomainThroughput | Bytes read throughput from Alluxio storage via domain socket |
+| cluster.BytesReadLocal | Total number of bytes read from local filesystem |
+| cluster.BytesReadLocalThroughput | Bytes read throughput from local filesystem |
+| cluster.BytesWrittenAlluxio | Total number of bytes written to Alluxio storage. This does not include UFS writes |
+| cluster.BytesWrittenAlluxioThroughput | Bytes write throughput to Alluxio storage |
+| cluster.BytesWrittenDomain | Total number of bytes written to Alluxio storage via domain socket |
+| cluster.BytesWrittenDomainThroughput | Throughput of bytes written to Alluxio storage via domain socket |
+
+* I/O to under storages
+
+| Metric Name | Description |
+|-----------------------------------|---------------------------------------------|
+| cluster.BytesReadUfsAll | Total number of bytes read from all Alluxio UFSes |
+| cluster.BytesReadUfsThroughput | Bytes read throughput from all Alluxio UFSes |
+| cluster.BytesWrittenUfsAll | Total number of bytes written to all Alluxio UFSes | 
+| cluster.BytesWrittenUfsThroughput | Bytes write throughput to all Alluxio UFSes |
+
+* Under storage RPCs
+
+For all th UFS RPCs (e.g. create file, delete file, get file status), 
+the timer metrics of each RPC will be recorded as well as the failure counters 
+if any.
+
+For example: `cluster.UfsOp<RPC_NAME>.UFS:<UFS_ADDRESS>` records the number of 
+UFS operation <RPC_NAME> ran on UFS <UFS_ADDRESS>
+
+### Master Metrics
+
+* Master summary information
+
+| Metric Name | Description |
+|-------------------------|-----------------------------------------------------|
+| Master.TotalPaths | Total number of files and directory in Alluxio namespace |
+| Master.UfsSessionCount-Ufs:<UFS_ADDRESS> | The total number of currently opened UFS sessions to connect to the given <UFS_ADDRESS> |
```

Comment:
backticks around `<UFS_ADDRESS>`?

Reply:
I am not exactly sure how backticks work with table using |||
It may be fine to leave values without backticks in the table.

Reply:
backticks should work within the table